runtime/doc/treesitter.txt

```diff
@@ -728,29 +727,28 @@ stop({bufnr})                                          *vim.treesitter.stop()*
 Lua module: vim.treesitter.language                  *lua-treesitter-language*
 
 add({lang}, {opts})                            *vim.treesitter.language.add()*
-    Asserts that a parser for the language {lang} is installed.
+    Add a parser by loading it
```

Comment:
```suggestion
    Load parser with name {lang}.
```
(Wasn't the plan to make the consistent switch `lang` -> `parser`?)

Reply:
It was, but I chickened out a bit.

The code funnels into `languagetree.lua` where using "parser" in place of "lang" gets confusing.

Upstream uses the term "language" and much of our low level stuff should align with that. Therefore, I think only top level stuff (e.g. an option) should use the Nvim specific "parser" term.

Where the exact boundary lies isn't clear to me.

Reply:
Makes sense; it is consistent here, which is the most important thing.